pyServer/GuestFishWrapper.py

```diff
@@ -98,6 +98,8 @@ def GetInspectMetadata(self, guestfish, device):
         return (osType[0], osDistribution[0], osProductName[0], osMountpoints)
 
     def CreateArchive(self, zipFilename, targetDir):
+        start_time = datetime.now()
+        self.rootLogger.info("Starting zip archiving.")
```

Comment:
Does this logged line benefit someone reviewing a production log?

Reply:
Yes, I think we need this. Previously there was no clear tracing to show where zip archiving started and ended. It wasn't until I found the time gap between the last operation before this ("Current working directory: /libguestfs") and the one after this ("Turning off KeepAlive messages..") that I realized there was a chunk of time used for archiving.